src/main/java/server/ServerHandler.java

```diff
@@ -79,109 +92,58 @@ private void executeCommand(Message message, Session mainSession) {
         try {
             switch (command) {
                 case "/list":
-                    for (Session session : sessionSet) {
-                        Message tempMessage = new Message(MessageType.MESSAGE, "[SERVER]", session.getUser().getName());
-                        mainSession.writeMessage(tempMessage);
-                    }
+                    Command commandList = new CommandList(sessionSet);
+                    commandList.execute(mainSession);
                     break;
 
                 case "/help":
-                    Message helpMessage = new Message(MessageType.MESSAGE, "[SERVER]", helpMessageString);
-                    mainSession.writeMessage(helpMessage);
+                    Command commandHelp = new CommandHelp();
+                    commandHelp.execute(mainSession);
```

Comment:
As a result you have multiple calls for `Command.execute`. Probably, we need to use a single variable for the command, set it in the switch's case to a corresponding implementation and after that call `Command.execute` once